test/gtest/log_test_helper.cpp

```diff
@@ -198,3 +226,32 @@ void TestLogFun(std::function<void(const miopenTensorDescriptor_t&,
     else
         ASSERT_FALSE(isSubStr(str, sub_str)) << "str     : " << str << "str_sub : " << sub_str;
 }
+
+void TestLogCmdFusion(std::function<void(const miopenFusionPlanDescriptor_t)> const& func,
+                      std::string env_var,
+                      std::string sub_str,
+                      bool set_env)
+{
+    // start capturing std::cerr
```

Comment:
Everywhere: In comments, please explain _why_ you are doing something, not _what_ you're doing.

If the source code is so unclear that comments are needed to understand what it does, then something is wrong (and requires some refactoring at least).

Reply:
Agree! comment should be why and code should be what.